### PR TITLE
[devtools] refactor js debugging docs

### PR DESCRIPTION
**Pull request by kaycebasques**
* Updates screenshots.
* Moves all of the debugging features to a single task-based "reference". They were previously scattered across a bunch of small docs. Uses a fairly consistent and much more concise style to describe each feature.
* Deprecates the old docs by putting warnings at top of each section, along with links to new content.
* Adds "deprecation" status to old docs in ToC.

### CODE REVIEW

File: src/content/en/tools/chrome-devtools/javascript/step-code.md [hunk 1]
```diff
@@ -204,9 +229,9 @@ To blackbox a file:
 
 3. Click **Add pattern**.
 
-4. In the **Pattern** textfield enter the filename pattern that you wish to 
-   exclude from your call stack. DevTools excludes any scripts that match the 
-   pattern. 
+4. In the **Pattern** textfield enter the filename pattern that you wish to
```
Comment (jpmedley) [inline src/content/en/tools/chrome-devtools/javascript/step-code.md]: The term is 'text box', two words.

File: src/content/en/tools/chrome-devtools/javascript/step-code.md [hunk 2]
```diff
@@ -28,6 +31,11 @@ After [setting breakpoints](breakpoints), return to the page and use it normally
 
 ## Stepping in action
 
+<aside class="warning">
+  <b>Warning:</b> This page is deprecated. See following section for up-to-date
```
Comment (jpmedley) [inline src/content/en/tools/chrome-devtools/javascript/step-code.md]: 'See following' to 'See the following'. Check throughout.

Comment (jpmedley) [inline src/content/en/tools/chrome-devtools/javascript/step-code.md]: I'm also wondering why you're not linking to the new section right here.

File: src/content/en/tools/chrome-devtools/javascript/step-code.md [hunk 3]
```diff
@@ -174,14 +193,20 @@ Call stack with anonymous functions:
...
 
 <!-- blackbox OR disable third-party code??? -->
 
 ### Blackbox third-party code
 
+<aside class="warning">
+  <b>Warning:</b> This page is deprecated. See following section for up-to-date
+  information: <a href="reference#blackbox">Ignore a script or pattern of
+  scripts</a>
+</aside>
```
Comment (jpmedley) [inline src/content/en/tools/chrome-devtools/javascript/step-code.md]: Warning: blah blah blah is sufficient to get the correct formatting in Devsite. Is there some reason you're going to the extra trouble of HTML?

File: src/content/en/tools/chrome-devtools/javascript/step-code.md (hunk 2 above)
Comment (kaycebasques) [inline src/content/en/tools/chrome-devtools/javascript/step-code.md]: Done

Comment (kaycebasques) [inline src/content/en/tools/chrome-devtools/javascript/step-code.md]: It does link to the new section?

File: src/content/en/tools/chrome-devtools/javascript/step-code.md (hunk 3 above)
Comment (kaycebasques) [inline src/content/en/tools/chrome-devtools/javascript/step-code.md]: I was using a custom callout in a another doc and then copy-pasted this over. No reason not to use `Warning: blah blah`

File: src/content/en/tools/chrome-devtools/javascript/step-code.md (hunk 1 above)
Comment (kaycebasques) [inline src/content/en/tools/chrome-devtools/javascript/step-code.md]: Thanks, will remember going forward. This is a deprecated page so I'm not going to bother with it.

**Comment by kaycebasques** [comment]
Thanks for the review, @jpmedley. I updated per your review.

**Pull request merged**